src/Sentry/Laravel/Features/Storage/Integration.php

```diff
@@ -17,26 +17,12 @@ class Integration extends Feature
 
     public function isApplicable(): bool
     {
-        return $this->isTracingFeatureEnabled(self::FEATURE_KEY)
-            || $this->isBreadcrumbFeatureEnabled(self::FEATURE_KEY);
+        // Since we only register the driver this feature is always applicable
+        return true;
     }
 
     public function setup(): void
     {
-        foreach (config('filesystems.disks') as $disk => $config) {
-            $currentDriver = $config['driver'];
-
-            if ($currentDriver === self::STORAGE_DRIVER_NAME) {
-                continue;
-            }
-
-            config([
-                "filesystems.disks.{$disk}.driver" => self::STORAGE_DRIVER_NAME,
-                "filesystems.disks.{$disk}.sentry_disk_name" => $disk,
-                "filesystems.disks.{$disk}.sentry_original_driver" => $config['driver'],
-            ]);
-        }
```

Comment:
I would love to see this preserved in a `public static function configureDisksWithSentryDriver(): void` in this class. The documentation could mention that it can be used to simplify the configuration change.

Reply:
We can follow up on this later. For now, we want to get a fix out ASAP.